eng/tools/GenerateFiles/Directory.Build.targets.in

```diff
@@ -36,6 +36,10 @@
       Condition=" (('$(ProjectTargetFrameworkIdentifier)' == '${DefaultNetCoreTargetFramework}') AND '${DefaultNetCoreTargetFramework}' == '$(ProjectTargetFrameworkVersion)') AND '$(TargetLatestDotNetRuntime)' != 'false' "
       RuntimeFrameworkVersion="${MicrosoftNETCoreAppRuntimeVersion}"
       TargetingPackVersion="${MicrosoftNETCoreAppRefVersion}" />
+
+    <PackageReference Include="Microsoft.Dotnet.ILCompiler"
+      Condition=" '$(PublishAot)' == 'true' "
+      Version="$(MicrosoftNETCoreAppRuntimeVersion)" />
```

Comment:
I'm curious, does it have to be `${MicrosoftNETCoreAppRuntimeVersion}`?

Reply:
Of course! :)